### PR TITLE
tox: Update tested python versions

### PR DESCRIPTION
**Pull request by okurz**
Keep 3.6 for openSUSE Leap/SLE. Use more recent versions available in
openSUSE Tumbleweed as well as ubuntu-latest.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -18,6 +18,7 @@ python =
     3.10: py310
     3.11: py311
     3.12: py312
+    3.13: py313
```
Comment (nicksinger) [inline tox.ini:21]: I guess this is not needed in [python-package.yaml](https://github.com/os-autoinst/openqa_review/pull/223/files#diff-99b88a98efd0677d898beb3aa7118fc2ad81b89c77a1abba84243ffb1d830fa8R14)?

Comment (okurz) [reply tox.ini:21]: No, python 3.13 is not fully released yet so this is just prepared for local execution or other environments even though not yet used in CI tests based on ubuntu.

**Review by perlpunk — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (sqozz) [reply tox.ini:21]: alright, got it thx

Comment (nicksinger) [reply tox.ini:21]: alright makes sense, thanks

**Review by nicksinger — APPROVED** [review]

**Pull request merged**